src/main/kotlin/ornn/domain/Player.kt

```diff
@@ -0,0 +1,27 @@
+package ornn.domain
+
+abstract class Player(private val name: String, private var cards: Cards) {
+    fun getName(): String {
+        return name
+    }
+
+    fun getCards(): Cards {
+        return cards
+    }
+
+    fun setCards(cards: Cards) {
+        this.cards = cards
+    }
+
+    fun takeCard(opCards: Cards) {
+        val card = opCards.removeLast()
+        cards.add(card)
+    }
+
+    fun isCardsSumMoreThanNum(num: Int): Boolean {
+        if (getCards().getScoreSum() > num) {
+            return true
+        }
+        return false
```

Comment:
```suggestion
        return cards.isBiggerScoreThan(num)
```
먼저 조건문에 따라 true와 false를 반환하는 메서드는 조건을 return하는 것으로 간단하게 줄일 수 있어요! 
그리고 현재 Player 본인의 프로퍼티를 사용하는 것이라 getCards()를 호출하지 않고 바로 cards를 사용해도 좋을 거 같아요ㅎㅎ
마지막으로 cards의 score를 꺼내와서 비교를 하는 대신에 메세지를 던지는 식으로 변경해보는 건 어떠신가요?